File: NotificationHubsSample.WinUsingAMS/NotificationHubsSample.WinUsingAMS.Shared/Model/Car.cs
using System;

namespace NotificationHubsSample.WinUsingAMS.Model
{
    /// <summary>
    /// Define the Car.
    /// </summary>
    public class Car
    {
        /// <summary>
        /// Gets or sets the name.
        /// </summary>
        /// <value>The name.</value>
        public string Name { get; set; }

        /// <summary>
        /// Gets or sets a value indicating whether this instance is electric.
        /// </summary>
        /// <value><c>true</c> if this instance is electric; otherwise, <c>false</c>.</value>
        public bool IsElectric { get; set; }

        /// <summary>
        /// Gets or sets the created at.
        /// </summary>
        /// <value>The created at.</value>
        public DateTimeOffset? CreatedAt { get; set; }

        /// <summary>
        /// Gets or sets a value indicating whether this <see cref="Car"/> is deleted.
        /// </summary>
        /// <value><c>true</c> if deleted; otherwise, <c>false</c>.</value>
        public bool Deleted { get; set; }

        /// <summary>
        /// Gets or sets the identifier.
        /// </summary>
        /// <value>The identifier.</value>
        public string Id { get; set; }

        /// <summary>
        /// Gets or sets the updated at.
        /// </summary>
        /// <value>The updated at.</value>
        public DateTimeOffset? UpdatedAt { get; set; }

        /// <summary>
        /// Gets or sets the version.
        /// </summary>
        /// <value>The version.</value>
        public byte[] Version { get; set; }
    }
}

File: src/NotificationHubsSample.Win81UsingAMS (VB)/Model/Car.vb
''' <summary>
''' Define the Car.
''' </summary>
Public Class Car
    ''' <summary>
    ''' Gets or sets the name.
    ''' </summary>
    ''' <value>The name.</value>
    Public Property Name As String

    ''' <summary>
    ''' Gets or sets a value indicating whether this instance is electric.
    ''' </summary>
    ''' <value><c>true</c> if this instance is electric; otherwise, <c>false</c>.</value>
    Public Property IsElectric As Boolean

    ''' <summary>
    ''' Gets or sets the created at.
    ''' </summary>
    ''' <value>The created at.</value>
    Public Property CreatedAt As System.Nullable(Of DateTimeOffset)

    ''' <summary>
    ''' Gets or sets a value indicating whether this <see cref="Car"/> is deleted.
    ''' </summary>
    ''' <value><c>true</c> if deleted; otherwise, <c>false</c>.</value>
    Public Property Deleted As Boolean

    ''' <summary>
    ''' Gets or sets the identifier.
    ''' </summary>
    ''' <value>The identifier.</value>
    Public Property Id As String

    ''' <summary>
    ''' Gets or sets the updated at.
    ''' </summary>
    ''' <value>The updated at.</value>
    Public Property UpdatedAt As System.Nullable(Of DateTimeOffset)

    ''' <summary>
    ''' Gets or sets the version.
    ''' </summary>
    ''' <value>The version.</value>
    Public Property Version As Byte()
End Class
